homeassistant/components/velbus/select.py

```diff
@@ -0,0 +1,50 @@
+"""Support for Velbus select."""
+from velbusaio.channels import SelectedProgram
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import VelbusEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Velbus select based on config_entry."""
+    await hass.data[DOMAIN][entry.entry_id]["tsk"]
+    cntrl = hass.data[DOMAIN][entry.entry_id]["cntrl"]
+    entities = []
+    for channel in cntrl.get_all("select"):
+        entities.append(VelbusSelect(channel))
+    async_add_entities(entities)
+
+
+class VelbusSelect(VelbusEntity, SelectEntity):
+    """Representation of a select option for velbus."""
+
+    _channel: SelectedProgram
+    _attr_entity_category = EntityCategory.CONFIG
```

Comment:
Are you sure to add this to config?

Reply:
this is debateable, but i think config is on its place, you can not change this from the modules itself, you need to do this via the config software